openlibrary/templates/search/authors.html

```diff
@@ -26,7 +26,7 @@ <h1>$_("Search Authors")</h1>
 </div>
 
 <div id="contentMeta">
-    $ results = get_results(q, offset=offset, limit=results_per_page, sort=sort)
+    $ results = get_results(q='', offset=offset, limit=results_per_page, sort=sort)
```

Comment:
This will _set_ the query to the empty string :P Effectively ignoring the user's query.

Reply:
Using ```strip``` function to removed the empty spaces and will also allow user's query.